package.json

```diff
@@ -44,6 +44,7 @@
     "node-fetch": "^2.6.1"
   },
   "devDependencies": {
+    "@cloudflare/workers-types": "^4.20230628.0",
```

Comment:
What do you think about just redefining the types inline, similar to https://github.com/michaelcpuckett/activity-kit/blob/2119ea8e95fde451d5057d7bd6d95330de34c15e/packages/types/lib/adapters/FetchPolyfill.d.ts? I think it's a little confusing to leak refs from the cloudflare package in source (even though it's only a devDependecy)